kroxylicious/src/main/java/io/kroxylicious/proxy/config/admin/PrometheusMetricsConfig.java

```diff
@@ -5,6 +5,9 @@
  */
 package io.kroxylicious.proxy.config.admin;
 
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@JsonSerialize
```

Comment:
something missed in the fidelity tests, this empty object couldn't be serialized back to YAML without an annotation